dasharo-compatibility/dasharo-tools-suite.robot

```diff
@@ -0,0 +1,150 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../variables.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     AND
+...                     Enable Network Boot
+...                     AND
+...                     Disable Secure Boot
+...                     AND
+...                     Upload And Mount DTS Flash ISO
```

Comment:
You have already developed `Download ISO And Mount As USB ` which is more generic and works on both QEMU and PiKVM.

Reply:
[Replaced](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/c25cb497706d8a74c04839a38cf75f2ad65e1a61#diff-4d0e5cbb7f43565e1c6904f6f0c660951a6f60f7d187a787aa23d8fcab380777R22). @macpijan should the DTS url and name be defined in each test, or rather alongside the keyword in the USB lib itself?